apps/src/templates/projects/SectionProjectsList.jsx

```diff
@@ -0,0 +1,89 @@
+import React, {Component, PropTypes} from 'react';
+import ProjectsList from './ProjectsList';
+import StudentFilterDropdown from './StudentFilterDropdown';
+import _ from 'lodash';
+import color from "../../util/color";
+
+const ALL_STUDENTS = '_all_students';
```

Comment:
This is defined in two places; maybe it should live in `StudentFilterDropdown` and get imported here.